CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## [2.0.0-beta2](https://www.github.com/googleapis/java-bigtable-hbase/compare/v2.0.0-beta1...v2.0.0-beta2) (2021-10-19)
+
+**Note: This beta release is a work-in-progress. For the latest stable release of java-bigtable-hbase, please refer to version [1.25.0](https://github.com/googleapis/java-bigtable-hbase/releases/tag/v1.25.0).**
+
+### Bug Fixes
+
+* fix user agent format ([#3288](https://www.github.com/googleapis/java-bigtable-hbase/issues/3288)) ([49ad39f](https://www.github.com/googleapis/java-bigtable-hbase/commit/49ad39f6ef5e5bd2c294202685816c9f2b61e74b))
+
+
+### Reverts
+
+* Revert "build(java): Introduce Native Image testing build script changes" ([#3292](https://www.github.com/googleapis/java-bigtable-hbase/issues/3292)) ([142a110](https://www.github.com/googleapis/java-bigtable-hbase/commit/142a1101b8891099a4222707a0112b56b48266f8))
+* Revert "deps: update dependency com.google.guava:guava ([#3261](https://www.github.com/googleapis/java-bigtable-hbase/issues/3261))" ([#3294](https://www.github.com/googleapis/java-bigtable-hbase/issues/3294)) ([c71f0b2](https://www.github.com/googleapis/java-bigtable-hbase/commit/c71f0b2527d99b5f5e6195c30c73167989ea68fd))
+
+
+### Miscellaneous Chores
+
+* bump next tag to 2.0.0-beta2 ([#3283](https://www.github.com/googleapis/java-bigtable-hbase/issues/3283)) ([8753c30](https://www.github.com/googleapis/java-bigtable-hbase/commit/8753c309004e50b208f72e01bdc9318349297254))
+
+
+### Dependencies
+
+* update dependency com.google.guava:guava ([#3261](https://www.github.com/googleapis/java-bigtable-hbase/issues/3261)) ([f90bfd1](https://www.github.com/googleapis/java-bigtable-hbase/commit/f90bfd13d05ce8846c079b840317aa55c42c4adf))
```

Comment:
Should we remove this change log since the version is reverted?

Reply:
good call, will fix